app/styles/_membership.less

```diff
@@ -31,7 +31,7 @@
     .col-name {
       min-width: 165px;
       padding: 0 5px 10px 0;
-      word-wrap: break-word;
+      word-break: break-all;
```

Comment:
@spadgett Yeah, `word-wrap: break-word;` doesn't get the job done with the non-hyphenated words.
@jwforres Mmm... I just forgot about it :)

Let me look @ the mixin quick and see if that does the job.